src/components/sidebar/helpers/generate-product-landing-nav-items.ts

```diff
@@ -51,7 +52,7 @@ export const generateProductLandingSidebarMenuItems = (
 			fullPath: `/${product.slug}/downloads`,
 		})
 	}
-	if (product.integrationsConfig.enabled) {
+	if (getIsEnabledProductIntegrations(product.slug)) {
```

Comment:
Naming tho...need to make this name better.

Reply:
`integrationsEnabledForProduct(product)` ?